tests/ui/cfg/true-false.rs

```diff
@@ -1,7 +1,6 @@
 //@ run-pass
 
 #![feature(link_cfg)]
-#![feature(cfg_boolean_literals)]
 
 #[cfg(true)]
```

Comment:
Question: this is somewhat pedantic, but is there a negative test for e.g.

- That *only* exactly `true`/`false` are accepted, *not* `r#true`/`r#false`.
- That *only* `true`/`false` is permitted, i.e. not `cfg(true = "")` or `cfg(true(false))`.
- Can this be used as an *inner* attribute on the crate root?

Reply:
For instance, it may be good to add a test for its behavior when used as an inner attribute on the crate-level e.g.

```rust
#![feature(cfg_boolean_literals)]
#![cfg(false)]

fn main() {}
```

does not compile because the whole file is `cfg(false)`'d out, whereas

```rs
#![feature(cfg_boolean_literals)]
#![cfg(true)]

fn main() {}
```

compiles.

(And maybe even `#![cfg_attr(true, cfg(true))]` etc. at crate-root.)

Reply:
Ah right. Is there a test case for accepting the raw literals? I'm not seeing it in the test diffs, although I might be completely missing it.

Reply:
I believe it's this one. https://github.com/rust-lang/rust/blob/493c38ba371929579fe136df26eccd9516347c7a/tests/ui/cfg/raw-true-false.rs

Reply:
Oh right, so only the `true`/`false` were gated. Makes sense.

Reply:
I think this is sufficiently addressed so resolving.